Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM google/dart:2.0.0 as build
+FROM google/dart:2.1.1 as build
```

Comment:
Should we also bump the version in the pubspec?

Reply:
The current range is `'>=1.23.0 <3.0.0'` so I don't think we need to update it.